.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: rust
 rust:
-  - nightly-2019-09-03
   - nightly
```

Comment:
should actually switch to stable, near-bindgen itself doesn't require nightly

Reply:
good point